<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="author" content="Edward Wallace" />
  <title>RNATagSeq Library Preparation Protocol</title>
  <style type="text/css">code{white-space: pre;}</style>
</head>
<body>
<div id="header">
<h1 class="title">RNATagSeq Library Preparation Protocol</h1>
<h2 class="author">Edward Wallace</h2>
<h3 class="date">9 February 2017</h3>
</div>
<p>This protocol is for RNA sequencing (RNASeq) library preparation using the RNATagSeq method of Shishkin, et al. (2015). <em>Simultaneous generation of many RNA-seq libraries in a single reaction.</em> Nature Methods, 12(4), 323–325. http://doi.org/10.1038/nmeth.3313 (Broad institute).</p>
<p>The protocol is adapted by Edward Wallace, 2015-7</p>
<pre><code>- increasing some volumes to make SPRI bead handling easier. 
- use AMPure XP beads as only SPRI beads
- use SuperAse-In as only RNase inhibitor 
- put a random barcoded on the read 2 (3Tr3) adaptor
- re-number the barcoded adaptors (T_xx) so that consecutive groups of 4 are colour-balanced
- redesign P7 index primers.</code></pre>
<p>Applies to batches of 4 to 32 samples.</p>
<p>The RNA-seqlopedia (http://rnaseq.uoregon.edu/) is a great background introduction to RNASeq.</p>
<h2 id="reagents">Reagents</h2>
<h3 id="wet-reagents">Wet reagents</h3>
<ul>
<li>Turbo DNase; Ambion/Applied Biosystems, Cat.# AM2239</li>
<li>FastAP Thermosensitive Alkaline Phosphatase; Thermo Scientific, Cat.# EF0651</li>
<li>RLT buffer (RNeasy Lysis Buffer); Qiagen, Cat.# 79216 (220mL)</li>
<li>T4 RNA Ligase 1; 30,000U/mL (0.5mL) Cat.# MO437M and comes with the following:
<ul>
<li>ATP(100mM) (make 10uL aliquots and store at -80°C, always use fresh aliquot)</li>
<li>PEG8000,50% in water</li>
<li>DMSO (100%); Sigma, Cat.# D8418-50ML for Molecular Biology</li>
</ul></li>
<li>SUPERase-IN 20U/uL (Ambion; AM2694; 2500U)</li>
<li>AffinityScript Multiple Temperature cDNA Synthesis Kit, 50 reaction; Agilent, Cat.# 200436. includes the
<ul>
<li>dNTPs, 10x RT buffer, RNase Block Ribonuclease Inhibitor (40U/uL)</li>
<li>10X AffinityScript RT buffer; Agilent, Cat.# 600100-52</li>
<li>RNA Clean &amp; ConcentratorTM-5 columns; Zymo Research, Cat. #R1015 (50 preps)</li>
</ul></li>
<li>SPRI beads:
<ul>
<li>SPRI beads / AMPure XP beads; Beckman/Agencourt, Cat.# A63881 (60mL). (mix well, make 1.5mL and 0.5mL aliquots, store at 4 °C). Original protocol recommended RNAClean XP beads, which are the same, just tested for no RNase activity.</li>
</ul></li>
<li>Ribo-Zero:
<ul>
<li>Bacteria: Ribo-ZeroTM Magnetic Gold Kit (Bacteria); Epicentre/Illumina, Cat.# MRZMB126</li>
<li>Human: Ribo-ZeroTM Magnetic Gold Kit (Human/Mouse/Rat); Epicentre/Illumina, Cat.# MRZG126</li>
<li>Yeast: Ribo-ZeroTM Magnetic Gold Kit (Yeast); Epicentre/Illumina, Cat.# MRZY1306</li>
</ul></li>
<li>Sodium hydroxide solution (5M); Sigma-Aldrich, Cat.# 656046</li>
<li>Glacial acetic acid (17.4M); Sigma-Aldrich, Cat.# ARK2183</li>
<li>Nuclease free water; Affymetrix Cat.#71786 100 ML or Ambion, Cat.# 4387936</li>
<li>1x low TE (10mM Tris pH 8.0, 0.1mM EDTA); Affymetrix Cat.# 75793</li>
<li>PfuUltra II Fusion HS DNA Polymerase; Agilent Cat.# 600670
<ul>
<li>10X PfuUltra II Reaction Buffer</li>
</ul></li>
</ul>
<h3 id="platestubescaps">Plates/Tubes/Caps:</h3>
<ul>
<li>Flat PCR 12-cap strips, Optically clear; USA Scientific, Cat.# 1400-3120</li>
<li>TempPlate No-skirt PCR Plates, 96 wells, 10 plates; USA Scientific, Cat.# 1402-9608</li>
<li>0.2 ml TempAssure PCR tube, attached flat cap, natural; USA Scientific, Cat.# 1402-8100</li>
<li>Wide-orifice pipette tips, 20uL and 200uL (for pipetting PEG-8000)</li>
<li>RNA Clean &amp; ConcentratorTM-5 kit; Zymo Cat.#R1015</li>
</ul>
<h3 id="instruments">Instruments:</h3>
<ul>
<li>2100 Electrophoresis Bioanalyzer Instrument; Agilent, Cat.# G2939AA
<ul>
<li>Agilent RNA 6000 Pico Kit; Agilent, Cat.# 5067-1513</li>
<li>Agilent DNA HS Kit; Agilent, Cat.# 5067-4626</li>
</ul></li>
<li>QIAvac 24 Plus Vacuum manifold (1-24 spin columns; Qiagen cat# 19413)</li>
<li>Thermocycler</li>
<li>10uL, 100uL, 200uL pipettes.</li>
<li>96R Ring Magnet Plate. Beckman/Agencourt Cat#. A29164 or Alpaqua Cat.# A001219</li>
<li>96-well Aluminum Cooler Blocks, Light Labs USA Cat#. A-7079</li>
</ul>
<h2 id="notes">Notes</h2>
<ul>
<li><strong>This is not a protocol where you can cut corners. To save time and anguish, plan thoroughly, work cleanly, QC your samples and intermediate steps.</strong></li>
<li>Label everything and lay out tubes neurotically to avoid pipette error.</li>
<li>Observe RNase-free sample prep, for steps 1-8, also neurotically. Clean your workspace and pipettes with 50% Ethanol and RNAse-Zap, use reserved RNase-free pipettes and RNAse-free (non-autoclaved) tips, ensure all tubes and reagents (including water) are RNase-free.</li>
<li>Avoid DNA contamination, <em>especially from other sequencing libraries</em> for steps 8+. Use filter tips for PCR reactions. Consider using separate pipettes and a laminar flow hood with UV decontamination.</li>
<li>Points where the in-process libraries may be frozen at -80 C while you pause, are indicated as <em>PAUSE POINT</em>. Don't take the risk of pausing elsewhere.</li>
<li>SPRI/XP beads take a bit of skill. &quot;2x SPRI&quot; refers to the ratio of beads to aqueous solution containing nucleic acids (NA); this ratio controls the size of NA retained (see http://core-genomics.blogspot.com/2012/04/how-do-spri-beads-work.html). Be patient; wait for binding and for magnetisation. Prepare fresh 80% ethanol solution every day or two; the concentration drifts over time due to evaporation (manufacturer recommends 70% EtOH). Some reactions (2nd ligation, step 11) happen on-bead so change the elution step. Manufacturer's protocol with tweaks is below; practice SPRI extraction on a DNA ladder or RNA ladder and ensure full recovery.
<ul>
<li>Add appropriate ratio of beads to sample in PCR tube, pipette up and down 10x gently.</li>
<li>Incubate beads and sample at room temperature for 15min to bind NA.</li>
<li>Place on 96R plate magnet for 5min, until solution is clear</li>
<li>Pipette out and discard clear solution (save this for troubleshooting)</li>
<li>Wash: Add 200uL fresh 80% EtOH without removing from magnet, incubate for 30sec, Pipette off clear solution.</li>
<li>Repeat 80% EtOH wash, use additional 10uL pipette to remove residual EtOH if necessary.</li>
<li>Let air dry for 2min. Apparently, drying too short is bad because residual ethanol interferes with downstream reactions, and too long is bad because beads may crack and/or bind hydrophobically, overly strongly, to your NA.</li>
<li>Elute: remove from magnet, pipette 40uL water or low TE buffer onto beads, pipette up and down 10X, replace on magnet and wait 5min (this is easiest with at least 40uL, hellish with 10uL).</li>
<li>Unless otherwise specified, take clear liquid to new tube, carefully not disturbing beads.</li>
</ul></li>
<li>For first ligation (step 5), set up RNA/barcoded adaptor in single tube or use the TempPlate No-skirt PCR Plates for batched samples (these rigid plates are easy to handle and can mix well by flicking by hand) – with a razor, cut a column (for 8 samples) or row (for 12 samples) and use as strip and cover with the Flat PCR 12-cap strips (these strips fit tightly on these plates and will not leak).</li>
<li>TempPlate No-skirt PCR Plates, and strips cut from them, are flexible and may spring up when caps are put on or off, flicking your sample across the bench. Avoid this by firmly seating plates/strips in a tube rack before manipulating the caps.</li>
<li><p>Barcoded adaptors are listed in appendix, requiring 5' Phosphate group (/5Phos/) and 3' blocking spacer (/3SpC3/). The combination to use should be chosen carefully: plan a spreadsheet listing the sample name, conditions, and which RNAtag adaptor (step 3) and P7 adaptor (step 14/17, PCR after pooling) will be used. In each set of sequencing libraries combined on an Illumina sequencing run, there must be at least 3 distinct nt in all of the initial 4 nt of the sequenced adaptor for the sequencing machine to recognize clusters properly. (e.g. ATTA, GCAC, CAGT work, ATTA, GTAC, CTGT don't due to the common T.)</p></li>
<li>Control oligos:
<ul>
<li>Tag33FAM<br />
5' - /5Phos/AG AAC GAT TAG ATC GGA AGA GCG TCG TGT A/36-FAM/ - 3'<br />
is a fluorescent positive control oligo for first ligation and sample pooling. Use in place of a barcoded adaptor, for one sample, in first ligation, step 5. Success is indicated if fluorescence appears in 200-500nt smear:</li>
<li>TagDNA+<br />
5'-TAC ACG ACG CTC TTC CGA TCT TCA GTC AGT ATG TAC GCG TAG CGC AGC GAG CGG CGG GTG GCC ACG TCG CGG CAC ACG CGG ATG GAC AGG ATC GGG CCG GCG GGC AGG AAC TTC TCG TAG AGC ATG GCC TCG GTC ACG TCG GCG TGC AGG TCG CCC ACG TAG AGC GAG GCC AGC GGG TAC CCC GGG CCG CTG GCG TTC AT - 3'<br />
is a 200bp oligo as positive control for 2nd ligation through PCR. Use 50 pmol (0.5uL of 100uM stock) in place of cDNA in control reaction in step 11. Its 3' end resembles a cDNA product for a library with Tag barcode CTGACTGA (distinct from Tag1-32, Tag33FAM), and the insert is &quot;random&quot; (elephant PABP 1st exon).</li>
</ul></li>
</ul>
<h2 id="procedure">Procedure:</h2>
<ol style="list-style-type: decimal">
<li>Control quality and quantity of RNA with Agilent Bioanalyzer
<ul>
<li>Check RNA quality by running on the Agilent Bioanalyzer<br />
</li>
<li>Place 0.5-20 ug of total RNA in a tube. For large numbers of samples, the input per sample could be reduced as to not exceed the maximum binding capacity of Zymo columns (10ug) or input of RiboZero (5ug) at step 5. Since about 25% of the input remains prior to 2nd adaptor ligation (step 11), less than 20ug total RNA per pool (end of step 6) is recommended. For smaller number of samples, higher input is recommended to ensure sufficient material remains after rRNA depletion.<br />
</li>
<li>Increase the volume to 30uL with Nuclease free water<br />
</li>
<li>Add 2uL of SUPERase-IN (20U/uL)<br />
</li>
<li>Final total volume = 32uL (25ng/uL)<br />
</li>
<li>Continue to next step or freeze at -80C until ready to process samples</li>
</ul>
<p><em>PAUSE POINT</em></p></li>
<li>Fragment RNA using 2x FastAP buffer
<ul>
<li>Add 8 uL of 10X FastAP buffer to 32 uL RNA from step 1 (up to 1 ug) and mix well.<br />
</li>
<li>Incubate on <em>preheated</em> thermal cycler for 3 min at 94°C.<br />
</li>
<li>If RNA is partially degraded (RIN&lt;7), fragment 3 min at 92°C, prevents over-fragmenting samples.<br />
</li>
<li>Place on cold block on ice.</li>
</ul></li>
<li>Digest DNA and repair RNA: Combination DNase/FastAP treatment
<ul>
<li><p>Make a DNase/FastAP master mix, 40uL per sample:</p>
<table>
<thead>
<tr class="header">
<th>Reagent (for 2X FastAP master)</th>
<th>Amount</th>
<th>Final</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>nuclease-free water</td>
<td>10 uL</td>
<td></td>
</tr>
<tr class="even">
<td>SUPERase-IN (20U/uL)</td>
<td>2 uL</td>
<td>20U</td>
</tr>
<tr class="odd">
<td>TURBO DNase (2U/uL)</td>
<td>8 uL</td>
<td>16U</td>
</tr>
<tr class="even">
<td>FastAP (1U/uL)</td>
<td>20 uL</td>
<td>20U</td>
</tr>
<tr class="odd">
<td>----------------------------------</td>
<td>--------</td>
<td>-------</td>
</tr>
<tr class="even">
<td>total</td>
<td>40 uL</td>
<td>2X</td>
</tr>
</tbody>
</table>
<ul>
<li>Mix well</li>
</ul></li>
<li>Aliquot 40 uL into each tube/well with the 40uL of fragmented RNA from step 1.<br />
</li>
<li>Mix well<br />
</li>
<li><p>Incubate on preheated thermal cycler for 30 min at 37°C</p></li>
</ul></li>
<li>Cleanup (2x SPRI) to remove enzymes and reaction buffer
<ul>
<li>Add 2.0x reaction volume of SPRI beads (160 uL) and capture RNA on beads:
<ul>
<li>Incubate at room temperature for 15min to bind RNA</li>
<li>Place on magnet for 5min, until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Add 200uL fresh 80% EtOH without removing from magnet, incubate for 30sec, Pipette off supernatant.</li>
<li>Repeat 80% EtOH wash. Let air dry for 2min</li>
</ul></li>
<li>Elute off beads with 24 uL nuclease free water</li>
<li>Take 5 uL of each sample and proceed to 1st ligation</li>
<li>QC: Save 1.2 uL from remaining RNA before addition of SUPERase-IN</li>
<li>Run 11 random samplings on Agilent RNA pico chip to check the fragmentation profile of each batch of 32</li>
<li>Add 1uL SUPERase-IN (20U/uL) to the remaining material and store at -80°C.
<ul>
<li>this can be used as backup if it is necessary to repeat process</li>
</ul></li>
</ul>
<p><em>PAUSE POINT</em></p></li>
<li>Ligate 3’ barcoded Adaptor: First Ligation (ssRNA/ssDNA)
<ul>
<li>Add 1 uL of barcoded RNATag adaptor (100 pmole = 1 uL of 100 uM) to 5 uL of dephosphorylated RNA</li>
<li>Heat at 70°C for 2 min, place in cold block on ice</li>
<li>Set up First Ligation master mix below NOTE:
<ul>
<li>All reagents except enzymes (-20°C ) should be stored at -80°C in single use aliquots and brought to room temp just before use</li>
<li>Make up mix <em>at room temp</em> so the reagents don’t start precipitating when combined (if DMSO is added directly into cold buffer it will precipitate)</li>
<li>Pipette very slowly with wide-opening/cut tips for accurate aspiration of PEG (very viscous)</li>
<li><p>When setting up mix for multiple reactions include 25% extra to account for pipetting error due to the viscosity</p>
<table>
<thead>
<tr class="header">
<th>Reagent (for Ligation master)</th>
<th>1 rxn</th>
<th>40 rxns</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>10× T4 RNA Ligase Buffer</td>
<td>2 uL</td>
<td>80 uL</td>
</tr>
<tr class="even">
<td>DMSO (100%)</td>
<td>1.8 uL</td>
<td>72 uL</td>
</tr>
<tr class="odd">
<td>ATP (100 mM)</td>
<td>0.2 uL</td>
<td>8 uL</td>
</tr>
<tr class="even">
<td>PEG 8000 (50%)</td>
<td>8 uL</td>
<td>320 uL</td>
</tr>
<tr class="odd">
<td>SUPERase-IN (20U/uL)</td>
<td>0.3 uL</td>
<td>12 uL</td>
</tr>
<tr class="even">
<td>----------------------------------</td>
<td>---------</td>
<td>---------</td>
</tr>
<tr class="odd">
<td>Total</td>
<td>12.3 uL</td>
<td>492 uL</td>
</tr>
</tbody>
</table></li>
<li>Mix really well by extensive vortexing tube since the solution is very viscous, then spin down briefly in microfuge</li>
</ul></li>
<li>Add 12.3 uL of ligation master mix to each tube/well containing 6 uL denatured RNA + adaptor.</li>
<li>Add 1.8 uL of T4 RNA Ligase 1 (30,000U/mL) to each tube/well. 20.1 uL reaction volume total.</li>
<li>Mix well <em>many</em> times; mix by flicking since the solution is very viscous</li>
<li>Incubate at 22°C (room temp) for 1 hour 30 minutes.</li>
</ul></li>
<li>Pool barcoded RNA: RLT buffer + Zymo column<br />
NOTE: At this point, multiple samples with distinct RNAtag adaptors will be pooled on the same spin column. Do not exceed 10ug RNA per pool, the maximum binding capacity of columns. Attempt to normalize the amounts (using your QC in step 4, or even 1) based on the amount of non-ribosomal RNA in each pool, or your other needs.
<ul>
<li>Add 60 uL of RLT buffer to each sample to inhibit ligase, and mix well (80 uL total)</li>
<li>Carefully add reactions to be pooled to a single lo-bind 1.5mL tube.</li>
<li>Pooling and clean up using Zymo Clean &amp; Concentrator-5 column - follow manufacturer’s <em>200nt</em> cut off protocol:
<ul>
<li>Add 2x reaction vol (160 uL=2x 80 uL <em>per sample</em>) of 1:1 binding buffer: EtOH (100%) / reaction</li>
<li>Add each pool to a single Zymo column.<br />
NOTE: When pooling &gt;700 uL onto Zymo column use a vacuum manifold then proceed to centrifugation steps according to the manual</li>
<li>Wash and spin 1 min 12,000 g, then discard flow-through, once with 400 uL RNA Prep buffer, once with 800 uL RNA Wash buffer, once with 400uL Wash buffer, spin another 1min with no buffer.</li>
</ul></li>
<li>Elute 2 times with 16 uL nuclease free water for a total volume of 32 uL<br />
NOTE: 2 elutions help improve recovery/yield of RNA</li>
<li>Save 2 uL for QC-Run on Agilent RNA pico chip to check quantity and fragmentation profile</li>
</ul>
<p><em>PAUSE POINT</em></p></li>
<li>Deplete ribosomal RNA with RiboZero (optional)
<ul>
<li>See Manufacturer's instructions Quick Protocol for Ribo-ZeroTM</li>
<li>End with 1.8x SPRI cleanup, elute into 14 uL water, save 2 uL for QC</li>
</ul></li>
<li>Synthesize First Strand cDNA
<ul>
<li>Take 12 uL rRNA depleted RNA (use all the material from Ribo-Zero)</li>
<li>Include negative control sample, 12 uL H2O.</li>
<li>Add 2 uL (50 pmoles) of AR2 primer (25 uM) 5’-TAC ACG ACG CTC TTC CGA T-3' AR2, 53% GC, 19bp.</li>
<li>Mix well</li>
<li>Heat the mixture to 70°C for 2 min and immediately place on cold block on ice</li>
<li>Make RT master mix
<ul>
<li><p>Add in order on ice</p>
<table>
<thead>
<tr class="header">
<th>Reagent (for RT master)</th>
<th>1 rxn</th>
<th>5 rxns</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>10× AffinityScript RT Buffer</td>
<td>2 uL</td>
<td>10 uL</td>
</tr>
<tr class="even">
<td>DTT (0.1M)</td>
<td>2 uL</td>
<td>10 uL</td>
</tr>
<tr class="odd">
<td>25mM dNTP Mix (25mM each)</td>
<td>0.8 uL</td>
<td>4 uL</td>
</tr>
<tr class="even">
<td>SUPERase-IN (20U/uL)</td>
<td>0.4 uL</td>
<td>2 uL</td>
</tr>
<tr class="odd">
<td>----------------------------------</td>
<td>---------</td>
<td>----------</td>
</tr>
<tr class="even">
<td>Total</td>
<td>5.2 uL</td>
<td>26 uL</td>
</tr>
</tbody>
</table></li>
<li>Mix well</li>
</ul></li>
<li>Add 5.2 uL of RT mix to the 14 uL rRNA depleted RNA + AR2 RTprimer on ice</li>
<li>Add 0.8 uL of AffinityScript RT Enzyme</li>
<li>Mix well and spin for 5 sec</li>
<li>Place in <em>preheated</em> (55 °C) incubator or thermocycler. Incubate at 55 °C for 55 minutes.</li>
</ul></li>
<li>Degrade RNA after reverse transcription<br />
NOTE: make fresh working stock solutions of NaOH and Acetic Acid (0.5M stock: 10uL of Glacial Acetic Acid at 17.4M, and 338uL of H2O).
<ul>
<li>Add 10% reaction vol. of 1M NaOH (2 uL) to each reaction</li>
<li>Incubate at 70 °C for 12 minutes</li>
<li>Neutralize with 4 uL of 0.5M Acetic Acid; mix well</li>
<li>Total volume = 26 uL</li>
</ul></li>
<li>Cleanup reverse transcription (2x SPRI) to remove enzyme, primers, and reaction buffer
<ul>
<li>Add 14 uL of sterile water to each reaction for a final volume of 40 uL</li>
<li>Transfer to new tubes (NaOH may start degrading tubes)</li>
<li>Add 2x reaction volume SPRI beads (80uL) to the sample in new tubes, and mix up/down 10x</li>
<li>Incubate at room temperature for 15min</li>
<li>Place on magnet for 5 min or until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Wash: Add 200 uL fresh 80% EtOH without removing from magnet and incubate for 30 sec. Pipette off and discard the EtOH.</li>
<li>Repeat 80% EtOH wash, and let air dry for 3min, remove from magnet.</li>
<li>Add 5 uL RNase/DNase free water to beads -- <em>KEEP BEADS AND TUBES, do not transfer, do not pause</em></li>
</ul></li>
<li><p>Ligate 3’ Universal Adaptor: Second Ligation (ssDNA/ssDNA) with beads<br />
NOTE: 3Tr3 adaptor: 5’-/5Phos/NNN NNA GAT CGG AAG AGC ACA CG TCTG-/3SpC3/ 3’, 55% GC, 22bp, 5’-Phos and 3’-C3 spacer (or ddC, or dye). Make 40 uM stock.</p>
<ul>
<li>Start a positive control reaction, 50 pmoles (0.5uL of 100uM stock) of TagDNA+ in 5uL sterile H20 in place of cDNA</li>
<li>Add 2 uL (80 pmoles) of 3Tr3 adaptor to cDNA</li>
<li>Heat at 75°C for 3 min; Place on cold block on ice</li>
<li>Make ligation reaction master mix (can be prepared ahead of time, at RT):</li>
<li>2nd Ligation Master Mix:
<ul>
<li>Mix in order</li>
</ul>
<table>
<thead>
<tr class="header">
<th>Reagent (for Ligation 2 master)</th>
<th>1 rxn</th>
<th>2.5 rxns</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>10× T4 RNA Ligase Buffer</td>
<td>2 uL</td>
<td>5 uL</td>
</tr>
<tr class="even">
<td>DMSO (100%)</td>
<td>0.8 uL</td>
<td>2 uL</td>
</tr>
<tr class="odd">
<td>ATP (100 mM)</td>
<td>0.2 uL</td>
<td>0.5 uL</td>
</tr>
<tr class="even">
<td>PEG 8000 (50%)</td>
<td>8.5 uL</td>
<td>21.3 uL</td>
</tr>
<tr class="odd">
<td>T4 RNA Ligase 1 (30,000U/mL)</td>
<td>1.5 uL</td>
<td>3.8 uL</td>
</tr>
<tr class="even">
<td>----------------------------------</td>
<td>---------</td>
<td>----------</td>
</tr>
<tr class="odd">
<td>Total</td>
<td>13.0 uL</td>
<td>32.6 uL</td>
</tr>
</tbody>
</table>
<ul>
<li>Mix really well by extensive vortexing tube since the solution is very viscous</li>
<li>Spin down briefly in microfuge</li>
</ul></li>
<li>Swirl the 7uL cDNA/beads/water with pipet tip, THEN add 13 uL ligation 2 master mix.</li>
<li>Mix well by pipetting up and down 20x or cap tubes and flick several times; solution is viscous</li>
<li>Quick spin (low speed centrifuge, to get everything to bottom of tube)</li>
<li>Incubate overnight at 22 °C</li>
</ul></li>
<li>Cleanup (2x SPRI) to remove adaptors
<ul>
<li>Bring reaction volume to 40uL by adding 20uL nuclease-free water.</li>
<li>Add 2x reaction volume SPRI beads (80uL) to the sample, and mix up/down 10x</li>
<li>Incubate at room temperature for 15min</li>
<li>Place on magnet for 5 min or until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Wash: Add 200 uL fresh 80% EtOH without removing from magnet and incubate for 30 sec. Pipette off discard the EtOH.</li>
<li>Repeat 80% EtOH wash, and let air dry for 3min</li>
<li>Elute DNA by adding 33 uL nuclease-free water, transfer to new tube.</li>
</ul></li>
<li>2nd Cleanup (1.5x SPRI) to remove the remaining adaptors
<ul>
<li>Add 1.5x reaction volume SPRI beads (50 uL) to the sample in new tubes, and mix up/down 10x</li>
<li>Incubate at room temperature for 15min</li>
<li>Place on magnet for 5 min or until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Wash: Add 200 uL fresh 80% EtOH without removing from magnet and incubate for 30 sec. Pipette off and discard the EtOH.</li>
<li>Repeat 80% EtOH wash, and let air dry for 3min</li>
<li>Elute DNA by adding 25 uL RNase/DNase free water, transfer to new tube.</li>
</ul>
<p><em>PAUSE POINT</em></p></li>
<li>TEST PCR Amplification to determine final cycle number<br />
NOTE: P5 primer: P5_RNATag, 5’-AAT GAT ACG GCG ACC ACC GAG ATC TAC ACT CTT TCC CTA CAC GAC GCT CTT CCG ATC T-3’, 52% GC, 58bp; standard DNA oligo. Make 100uM stock and 12.5uM working stock.
<ul>
<li>Set up a test PCR using 5 uL of ss cDNA sample and 9-12 cycles of PCR (based on experience with pool of 16 reactions, each starting with ~400ng total RNA)</li>
<li>Include a negative control (water) for each primer set</li>
<li>Make PCR Master Mix (4 rxns=2 libraries, +ve ctrl, -ve ctrl):
<ul>
<li>Add in order:</li>
</ul>
<table>
<thead>
<tr class="header">
<th>Reagent (for PCR master mix)</th>
<th>1 rxn</th>
<th>4 rxns</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Water, PCR-clean</td>
<td>14.3 uL</td>
<td>57.2 uL</td>
</tr>
<tr class="even">
<td>10X Pfu Ultra II Buffer</td>
<td>2.5 uL</td>
<td>10 uL</td>
</tr>
<tr class="odd">
<td>dNTP mix (10mM each)</td>
<td>0.7 uL</td>
<td>2.8 uL</td>
</tr>
<tr class="even">
<td>P5 primer (P5_RNATag, 12.5 uM)</td>
<td>1 uL</td>
<td>4 uL</td>
</tr>
<tr class="odd">
<td>---------------------------------</td>
<td>---------</td>
<td>---------</td>
</tr>
<tr class="even">
<td>Total</td>
<td>18.5 uL</td>
<td>74 uL</td>
</tr>
</tbody>
</table>
<ul>
<li>Mix well</li>
<li>Aliquot 18.5 uL / sample into PCR tubes</li>
</ul></li>
<li>Add 1 uL of appropriate P7 index primer (12.5uM) to each well</li>
<li>Add 5 uL of ss cDNA from step 11, or water (for negative control)</li>
<li>Add 0.5 uL of Pfu Ultra II Polymerase.</li>
<li>Mix well and aliquot 8 ul into each of 3 tubes</li>
<li>Place each in a thermal cycler with cycling conditions:
<ul>
<li>start: 98°C, 3min</li>
<li>cycle: 9, 12, 15 cycles (for test PCR) 98°C, 30sec; 55°C, 30sec; 70°C, 30sec</li>
<li>end: 70°C, 2min; 4°C, hold</li>
</ul></li>
</ul></li>
<li>Cleanup (1.5x SPRI) to remove reaction buffer and PCR primers:
<ul>
<li>increase reaction to 40uL with sterile water</li>
<li>Add 1.5x reaction volume SPRI beads (60 uL) to the sample in new tubes, and mix up/down 10x</li>
<li>Incubate at room temperature for 15min</li>
<li>Place on magnet for 5 min or until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Wash: Add 200 uL fresh 80% EtOH without removing from magnet and incubate for 30 sec. Pipette off and discard the EtOH.</li>
<li>Repeat 80% EtOH wash, and let air dry for 3min</li>
<li>Elute off beads with 10 uL 1x low TE (10 mM Tris, 0.1M EDTA)</li>
</ul></li>
<li>QC test PCR amplification on Agilent DNA HS chip
<ul>
<li>Based on test results change the cycle number, if necessary, and set up more reactions to provide enough material to send for sequencing</li>
<li>UChicago functional genomics core asks for ~15 uL of 10 nM library; aim for at least 25 uL = 0.25 pmol = 60 ng of 400nt dsDNA (~250 kDa).</li>
<li>To pass QC, library should have smooth profile 200-500nt long; visible single bands, or a &quot;shoulder&quot; of larger products, indicate PCR artefacts.</li>
</ul>
<p><em>PAUSE POINT</em></p></li>
<li>PCR for Sequencing library
<ul>
<li>Choose the optimal PCR cycle # based on Bioanalyzer QC of test (step 15).</li>
<li>Include a negative control (water) for each primer set</li>
<li>Make PCR Master Mix (3 rxns=2 libraries, half size +ve ctrl, half size -ve ctrl):
<ul>
<li>Add in order:</li>
</ul>
<table>
<thead>
<tr class="header">
<th>Reagent (for PCR master mix)</th>
<th>1 rxn</th>
<th>3 rxns</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Water, PCR-clean</td>
<td>28.6 uL</td>
<td>85.8 uL</td>
</tr>
<tr class="even">
<td>10X Pfu Ultra II Buffer</td>
<td>5 uL</td>
<td>15 uL</td>
</tr>
<tr class="odd">
<td>dNTP mix (10mM each)</td>
<td>1.4 uL</td>
<td>4.2 uL</td>
</tr>
<tr class="even">
<td>P5 primer (P5_RNATag, 12.5 uM)</td>
<td>2 uL</td>
<td>6 uL</td>
</tr>
<tr class="odd">
<td>---------------------------------</td>
<td>---------</td>
<td>----------</td>
</tr>
<tr class="even">
<td>Total</td>
<td>37 uL</td>
<td>111 uL</td>
</tr>
</tbody>
</table>
<ul>
<li>Mix well</li>
<li>Aliquot 37 uL / sample into PCR tubes</li>
</ul></li>
<li>Add 2 uL of appropriate P7 index primer to each well</li>
<li>Add 10 uL of ss cDNA from step 11, or water (for negative control)</li>
<li>Add 1 uL of Pfu Ultra II Polymerase.</li>
<li>Mix well and aliquot 10 ul into each of 5 wells of a 96-well plate (amplification is apparently more robust in smaller volumes), cap.</li>
<li>Place each in a thermal cycler with the cycling conditions:
<ul>
<li>start: 98°C, 3min</li>
<li>cycle: # determined from test PCR 98°C, 30sec; 55°C, 30sec; 70°C, 30sec</li>
<li>end: 70°C, 2min; 4°C, hold</li>
</ul></li>
</ul></li>
<li>Cleanup (1.5x SPRI) to remove reaction buffer and PCR primers:
<ul>
<li>Pool PCR reaction (50 uL)</li>
<li>Add 1.5x reaction volume SPRI beads (75 uL) to the sample in new tubes, and mix up/down 10x</li>
<li>Incubate at room temperature for 15min</li>
<li>Place on magnet for 5 min or until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Wash: Add 200 uL fresh 80% EtOH without removing from magnet and incubate for 30 sec. Pipette off and discard the EtOH.</li>
<li>Repeat 80% EtOH wash, and let air dry for 3min</li>
<li>Elute off beads with 50 uL water and transfer to new tubes.</li>
</ul></li>
<li>Final Cleanup (0.8x SPRI) to remove remaining PCR primers:
<ul>
<li>Add 0.8x reaction volume SPRI beads (40 uL) to the sample in new tubes, and mix up/down 10x</li>
<li>Incubate at room temperature for 15min</li>
<li>Place on magnet for 5 min or until solution is clear</li>
<li>Pipette out and discard clear solution</li>
<li>Wash: Add 200 uL fresh 80% EtOH without removing from magnet and incubate for 30 sec. Pipette off and discard the EtOH.</li>
<li>Repeat 80% EtOH wash, and let air dry for 3min</li>
<li>Elute off beads with 25 uL 1x low TE (10 mM Tris, 0.1M EDTA)</li>
</ul></li>
<li>Proceed to sequence
<ul>
<li>Check quantity/quality on Bioanalyzer with Agilent DNA HS chip as step 16</li>
<li>Submit to genomics core for sequencing.</li>
<li>If desired to combine multiple libraries PCR'd separately, they must have different barcoded P7 primers. Amounts of sequenceable material must be carefully measured to ensure even coverage across libraries, e.g. with the KAPA biosystems kit (Cat. # KK4824). The genomics core can do this for a fee.</li>
</ul></li>
</ol>
<h2 id="appendix-barcode-tag-oligos">Appendix: Barcode Tag Oligos</h2>
<pre><code>- id = name of oligo.
- bc_read = barcode read (5&#39;-3&#39;), including trailing &quot;T&quot;
- oligo_seq = sequence of oligo to order (add 5&#39;Phos and 3&#39; Spacer)
- bc_seq = barcode sequence within oligo sequence
- old_id = id/name of oligo in original version of protocol (Shishkin et al).</code></pre>
<p>These are numbered so that blocks of 8 (1-8,9-16,etc) AND paired blocks of 4 (1-4 + 9-12, etc.) have at least 3 different nucleotides at every position.</p>
<pre><code>|-----|-----------|--------------------------------|----------|--------|
|id   | bc_read   | oligo_seq                      | bc_seq   | old_id |
|-----|-----------|--------------------------------|----------|--------|
|T_01 | CAACACTTT | AAAGTGTTGAGATCGGAAGAGCGTCGTGTA | AAGTGTTG | Tag09  |
|T_02 | CATCTGTAT | ATACAGATGAGATCGGAAGAGCGTCGTGTA | TACAGATG | Tag12  |
|T_03 | CTGGTTCAT | ATGAACCAGAGATCGGAAGAGCGTCGTGTA | TGAACCAG | Tag18  |
|T_04 | GGTACCGGT | ACCGGTACCAGATCGGAAGAGCGTCGTGTA | CCGGTACC | Tag27  |
|T_05 | AGTTAGAGT | ACTCTAACTAGATCGGAAGAGCGTCGTGTA | CTCTAACT | Tag06  |
|T_06 | GCGAGTTGT | ACAACTCGCAGATCGGAAGAGCGTCGTGTA | CAACTCGC | Tag25  |
|T_07 | GTCCATATT | AATATGGACAGATCGGAAGAGCGTCGTGTA | ATATGGAC | Tag29  |
|T_08 | CGTGGTAAT | ATTACCACGAGATCGGAAGAGCGTCGTGTA | TTACCACG | Tag17  |
|T_09 | CGCCAGACT | AGTCTGGCGAGATCGGAAGAGCGTCGTGTA | GTCTGGCG | Tag16  |
|T_10 | AAGTGCCGT | ACGGCACTTAGATCGGAAGAGCGTCGTGTA | CGGCACTT | Tag02  |
|T_11 | AGAGGACCT | AGGTCCTCTAGATCGGAAGAGCGTCGTGTA | GGTCCTCT | Tag05  |
|T_12 | ACAATCTCT | AGAGATTGTAGATCGGAAGAGCGTCGTGTA | GAGATTGT | Tag04  |
|T_13 | CGACTTGGT | ACCAAGTCGAGATCGGAAGAGCGTCGTGTA | CCAAGTCG | Tag14  |
|T_14 | ATGTTGTAT | ATACAACATAGATCGGAAGAGCGTCGTGTA | TACAACAT | Tag08  |
|T_15 | CTTGGGCCT | AGGCCCAAGAGATCGGAAGAGCGTCGTGTA | GGCCCAAG | Tag20  |
|T_16 | GTTAGAAAT | ATTTCTAACAGATCGGAAGAGCGTCGTGTA | TTTCTAAC | Tag32  |
|T_17 | GTGGCTGCT | AGCAGCCACAGATCGGAAGAGCGTCGTGTA | GCAGCCAC | Tag31  |
|T_18 | CGATCCAGT | ACTGGATCGAGATCGGAAGAGCGTCGTGTA | CTGGATCG | Tag15  |
|T_19 | GTACCGAGT | ACTCGGTACAGATCGGAAGAGCGTCGTGTA | CTCGGTAC | Tag28  |
|T_20 | CACGATGAT | ATCATCGTGAGATCGGAAGAGCGTCGTGTA | TCATCGTG | Tag11  |
|T_21 | GCCGAGGGT | ACCCTCGGCAGATCGGAAGAGCGTCGTGTA | CCCTCGGC | Tag24  |
|T_22 | AATAATGTT | AACATTATTAGATCGGAAGAGCGTCGTGTA | ACATTATT | Tag03  |
|T_23 | ATAATTCTT | AAGAATTATAGATCGGAAGAGCGTCGTGTA | AGAATTAT | Tag07  |
|T_24 | GCCCTCCGT | ACGGAGGGCAGATCGGAAGAGCGTCGTGTA | CGGAGGGC | Tag23  |
|T_25 | CCGCGGGAT | ATCCCGCGGAGATCGGAAGAGCGTCGTGTA | TCCCGCGG | Tag13  |
|T_26 | CTGTAGGGT | ACCCTACAGAGATCGGAAGAGCGTCGTGTA | CCCTACAG | Tag19  |
|T_27 | AAGACGGGT | ACCCGTCTTAGATCGGAAGAGCGTCGTGTA | CCCGTCTT | Tag01  |
|T_28 | GGCCGGTAT | ATACCGGCCAGATCGGAAGAGCGTCGTGTA | TACCGGCC | Tag26  |
|T_29 | GATGGCTCT | AGAGCCATCAGATCGGAAGAGCGTCGTGTA | GAGCCATC | Tag21  |
|T_30 | GCAGTTACT | AGTAACTGCAGATCGGAAGAGCGTCGTGTA | GTAACTGC | Tag22  |
|T_31 | GTCTCCCAT | ATGGGAGACAGATCGGAAGAGCGTCGTGTA | TGGGAGAC | Tag30  |
|T_32 | CAAGTGATT | AATCACTTGAGATCGGAAGAGCGTCGTGTA | ATCACTTG | Tag10  |
|-----|-----------|--------------------------------|----------|--------|</code></pre>
<h2 id="appendix-p7-barcode-index-oligos">Appendix: P7 Barcode-index Oligos</h2>
<p>Newly designed by EW in December 2016.</p>
<pre><code>| Name  | P7 Enrichment Primer Sequence (5&#39; --&gt; 3&#39;) with barcode             | barcodes (BC) | BC READ (reverse complement) | 
|-------|--------------------------------------------------------------------|---------------|------------------------------| 
| PBI_1 | CAAGCAGAAGACGGCATACGAGATGAGTGTGCGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | GAGTGTGC      | GCACACTC                     |
| PBI_2 | CAAGCAGAAGACGGCATACGAGATTGAGAACTGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | TGAGAACT      | AGTTCTCA                     |
| PBI_3 | CAAGCAGAAGACGGCATACGAGATGAGGACTGGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | GAGGACTG      | CAGTCCTC                     |
| PBI_4 | CAAGCAGAAGACGGCATACGAGATATTACGAAGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | ATTACGAA      | TTCGTAAT                     |
| PBI_5 | CAAGCAGAAGACGGCATACGAGATAGACTATGGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | AGACTATG      | CATAGTCT                     |
| PBI_6 | CAAGCAGAAGACGGCATACGAGATGTCAGTATGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | GTCAGTAT      | ATACTGAC                     |
| PBI_7 | CAAGCAGAAGACGGCATACGAGATTCGATGTAGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | TCGATGTA      | TACATCGA                     |
| PBI_8 | CAAGCAGAAGACGGCATACGAGATCGTTACGTGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT | CGTTACGT      | ACGTAACG                     |
|-------|--------------------------------------------------------------------|---------------|------------------------------| </code></pre>
</body>
</html>
